internal/compiler-bridge-test/src/test/scala/xsbt/DependencySpecification.scala

```diff
@@ -112,6 +112,25 @@ class DependencySpecification extends UnitSpec {
     assert(inheritance("B") === Set.empty)
   }
 
+  it should "extract class dependency from 'classOf' literal" in {
```

Comment:
Could you also add a scripted (integration) test?

Reply:
Do you mean adding a test a long side [this one](https://github.com/sbt/zinc/blob/1.x/zinc/src/sbt-test/source-dependencies/check-dependencies/test#L3)?

Reply:
I mean adding a test like that one but that would fail compilation if it this fix wasn’t  present, and that would succeed after the patch.

Reply:
done